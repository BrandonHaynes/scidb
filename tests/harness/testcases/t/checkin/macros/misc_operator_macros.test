--setup

--test
#
#   1. Test that it works in both AFL and AQL
#
op_count ( filter ( join ( op_now() AS FIRST, op_now() AS SECOND ), FIRST.when = SECOND.when ))
op_count ( filter ( join ( op_now() AS FIRST, op_now() AS SECOND ), FIRST.when <> SECOND.when ))
--aql SELECT COUNT(*) FROM op_now() AS A, op_now() AS B WHERE A.when = B.when

--cleanup
