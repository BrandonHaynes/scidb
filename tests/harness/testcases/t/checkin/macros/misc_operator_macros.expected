{i} count
{0} 1

{i} count
{0} 0

{i} COUNT
{0} 1

